cli/lib/errors.js

```diff
@@ -33,6 +33,15 @@ const missingApp = {
   `,
 }
 
+const nonZeroExitCodeXvfb = {
+  description: 'XVFB exited with a non zero exit code.',
+  solution: stripIndent`
+    There was a problem spawning Xvfb.
+
+    This is likely a problem with your system, permissions, or installation of Xvfb.
```

Comment:
Is there no suggestion for where they should go from here? Maybe it's just the wording, but it seems like a dead end.